app/sidero-controller-manager/config/crd/bases/metal.sidero.dev_servers.yaml

```diff
@@ -25,6 +25,10 @@ spec:
       jsonPath: .spec.accepted
       name: Accepted
       type: boolean
+    - description: indicates if the server is paused
```

Comment:
I don't know if `paused` is the right word, as it still allows server  to wiped, deallocated, etc. 

It just stops the server from being allocated.

@Ulexus ?